init-interfaces.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -ex
+
+if [[ ! -f /boot/mac_addresses ]] ; then
+  echo "no mac address configuration file found .. exiting"
+  exit 1
+fi
+
+primary_mac=$(cat /boot/mac_addresses | awk -F= '/PRIMARY_MAC/ {print $2}')
+secondary_mac=$(cat /boot/mac_addresses | awk -F= '/SECONDARY_MAC/ {print $2}')
+
+default_device=""
+secondary_device=""
+default_connection_name=""
+secondary_connection_name=""
+
+for dev in $(nmcli device status | awk '/ethernet/ {print $1}'); do
+  dev_mac=$(nmcli -g GENERAL.HWADDR dev show $dev | sed -e 's/\\//g' | tr '[A-Z]' '[a-z]')
+  case $dev_mac in
+    $primary_mac)
+      default_device=$dev
+      default_connection_name=$(nmcli -g GENERAL.CONNECTION dev show $dev)
+      ;;
+    $secondary_mac)
+      secondary_device=$dev
+      secondary_connection_name=$(nmcli -g GENERAL.CONNECTION dev show $dev)
+      ;;
+    *)
+      ;;
+   esac
+done
+
+echo -e "default dev: $default_device ($default_connection_name)\nsecondary dev: $secondary_device ($secondary_connection_name)"
+
+nmcli con up "$default_connection_name"
```

Comment:
It does not need to be handled in this PR, but: Any plans how to make it so this configuration does not conflict with day-2 configuration of the bridge and bonding? I'm thinking about

* What if this runs on day-1, then knmstate configures the rest, then the host reboots. Will this script work?
* Can we prevent this from running on every boot? I would bridge and bonding configured by knmstate to stay on the host once configured

Reply:
I tried to address this by setting the script to `Before=kubelet.service` on MCO. we will need to check this when we work on the bare metal

Reply:
Ok, generally speaking the interfaces script should not collide due to the `Before=kubelet.service` on MCO. 

Still, after reboot it will re-run, and might wreck the current knmstate thing.. 

so I suggest we set the interfaces up/down if the connections folder is empty. what do you say @phoracek ?

Reply:
The ordering sounds good +2.

It makes me uneasy that we will have a dependency on day-2 in this day-1 script, but I don't see a way around it.

Instead of the whole folder, could we check specifically for interface of a specific name? Then we can render the script parametrizing it so it does not run if there is interface `brcnv` in the system, maybe?

I suggest we do it in a follow-up PR. In this one, let's assume network config is volatile, including OVS database.